Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM openjdk:8u121-jre 
-MAINTAINER Kieran Wardle <kieran.wardle@ons.gov.uk>
-ARG jar
-VOLUME /tmp
-COPY $jar casesvc.jar
-RUN sh -c 'touch /casesvc.jar'
-ENV JAVA_OPTS=""
-ENTRYPOINT [ "sh", "-c", "java -jar /casesvc.jar" ]
+FROM openjdk:8-jre
+
+COPY target/casesvc*.jar /opt/casesvc.jar
+
```

Comment:
There is an edge case here - if you run 'mvn install' with no 'clean' after changing the pom version you'll end up with multiple files matching casesvc* - what's the docker behaviour in this case? Not sure there's a way around it though ...